nailgun/entities.py

```diff
@@ -6833,6 +6833,92 @@ def update_payload(self, fields=None):
         }
 
 
+class Snapshot(
+        Entity,
+        EntityCreateMixin,
+        EntityDeleteMixin,
+        EntityReadMixin,
+        EntitySearchMixin,
+        EntityUpdateMixin):
+    """A representation of a Snapshot entity.
+       Foreman_snapshot as mentioned in the plugin:
+       https://github.com/ATIX-AG/foreman_snapshot_management
```

Comment:
We recently decided to have example usage as part of the code. Eg: https://github.com/SatelliteQE/nailgun/blob/c36d8c20862e87bf6975bd48ac1ca40a9e634eaa/nailgun/entities.py#L470
It would be great if we can have it, Non-blocking comment though, As I am yet to update the documentation regarding this.

Reply:
@san7ket  I have added example usage for the Snapshot class.